apm-agent-plugins/apm-spring-webmvc-plugin/src/test/java/co/elastic/apm/agent/spring/webmvc/ExceptionHandlerInstrumentationWithExceptionResolverTest.java

```diff
@@ -43,13 +42,12 @@
 public class ExceptionHandlerInstrumentationWithExceptionResolverTest extends AbstractExceptionHandlerInstrumentationTest {
 
     @Test
-    @Ignore
     public void testCallApiWithExceptionThrown() throws Exception {
         ResultActions resultActions = this.mockMvc.perform(get("/exception-resolver/throw-exception"));
         MvcResult result = resultActions.andReturn();
         MockHttpServletResponse response = result.getResponse();
 
-        assertExceptionCapture(ExceptionResolverRuntimeException.class, response, 200, "", "runtime exception occured");
+        assertExceptionCapture(1, 2, ExceptionResolverRuntimeException.class, response, 200, "", "runtime exception occured");
```

Comment:
This is a problem... 
Why do we get two? I think this is what we try to avoid...